keylime/cloud_verifier_tornado.py

```diff
@@ -631,6 +631,7 @@ def post(self) -> None:
                     if not runtime_policy_name and not runtime_policy:
                         logger.info("IMA policy data not provided with request! Using default empty IMA policy.")
                         runtime_policy = json.dumps(cast(Dict[str, Any], ima.EMPTY_RUNTIME_POLICY))
+                        ima.trim_runtime_policy(runtime_policy, False)
```

Comment:
The code the convert the policy should not be out of the creator (create_policy.py, ima.py) or converter (convert_runtime_policy.py). 

I mean, maybe the next change of policy is not about trimming, but about extending, or translating. IMHO this piece should take the policy as such and not do any conversion.

Reply:
The problem in this case is that the ima.EMPTY_RUNTIME_POLICY unnecessarily is version 2 even though it only has an empty reject list. And then it may get written into the DB and prevent a should prevent a downgrade of the DB since we now have a v2 policy in the DB that no prior version of keylime knows how to deal with (keylime should reject using it since it wouldn't behave properly since it doesn't know what was added in v2).